contracts/wrapper/NameWrapper.sol

```diff
@@ -657,23 +645,58 @@ contract NameWrapper is
         emit NameWrapped(node, name, wrappedOwner, fuses);
     }
 
+    function _addLabelAndWrap(
+        bytes32 parentNode,
+        string memory label,
+        address newOwner,
+        uint96 _fuses
+    ) internal returns (bytes32 node, bytes32 labelhash) {
+        labelhash = keccak256(bytes(label));
+        node = _makeNode(parentNode, labelhash);
```

Comment:
Can we pass this in as an argument to avoid calculating it again?

Reply:
Done